src/main/java/me/jellysquid/mods/sodium/client/render/chunk/DefaultChunkRenderer.java

```diff
@@ -138,15 +138,18 @@ private static void addDrawCommands(MultiDrawBatch batch, long pMeshData, int ma
         final var pElementCount = batch.pElementCount;
 
         int size = batch.size;
-
+        int maxIndex = batch.maxElements;
```

Comment:
Why did you inline this?

Reply:
I don't believe this is faster. The original code is very simple and only accounts for a very small part of the runtime (<0.5%) according to YourKit Profiler...

It also doesn't behave the same way. Previously, only visible slices would be considered when calculating the maximum element index, but this considers every slice.

Reply:
good point i will revert this